src/fidesops/ops/schemas/email/email.py

```diff
@@ -14,7 +14,7 @@ class EmailServiceType(Enum):
     MAILGUN = "mailgun"
 
 
-class EmailActionType(Enum):
+class EmailActionType(str, Enum):
```

Comment:
Is this intentional? The type change?

Reply:
Yes, this enables the Enum to serialise into JSON in order to be passed into the Celery queues

Reply:
ah okay!